build/__init__.py

```diff
@@ -0,0 +1,3 @@
+import os
+
+PACKAGE_ROOT = os.path.dirname(os.path.dirname(__file__))
```

Comment:
Defining a global here and importing it in __main__ causes `python build` to fail. The main purpose of `__main__` is to be able to run the command without the `-m` option.

Reply:
May I interest you in the following code?

```python
if not __package__:
    import os, sys, runpy

    dir, module = os.path.split(os.path.dirname(os.path.realpath(__file__)))
    sys.path[0] = dir
    sys.exit(runpy._run_module_as_main(module, alter_argv=False) and 0)

# rest of __main__.py
```